state/factory/workingset.go

```diff
@@ -319,7 +321,11 @@ func (ws *workingSet) validateNonce(blk *block.Block) error {
 	}
 	// Verify each account's Nonce
 	for srcAddr, receivedNonces := range accountNonceMap {
-		confirmedState, err := accountutil.AccountState(ws, srcAddr)
+		addr, err := address.FromString(srcAddr)
```

Comment:
`accountNonceMap's`  key is `caller.String()`, which is a valid address, so can do
```
addr, _ := address.FromString(srcAddr)
```

Reply:
done